packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -12,6 +12,10 @@ const transformRuntime = require("../");
 const runtimeVersion = require("@babel/runtime/package.json").version;
 const corejs2Definitions = require("../lib/runtime-corejs2-definitions").default();
 const corejs3Definitions = require("../lib/runtime-corejs3-definitions").default();
+const outputFileSync = function(filePath, data) {
+  fs.mkdirSync(path.dirname(filePath), { recursive: true });
```

Comment:
This only supports node 10, I'm wondering why it doesn't file on Travis with node 6 and 8 :thinking:

Reply:
Because it targets to Babel 8? 🤓

Reply:
🙃